2.13.1 does not build with nix

```
$ git checkout 2.13.1
$ nix-build
...
[160 of 186] Compiling Chainweb.CutDB   ( src/Chainweb/CutDB.hs, dist/build/Chainweb/CutDB.o, dist/build/Chainweb/CutDB.dyn_o )

src/Chainweb/CutDB.hs:355:5: error:
    Variable not in scope:
      deleteRangeRocksDb
        :: RocksDbTable (CutHeight, BlockWeight, CutId) CutHashes
           -> (Maybe a0, Maybe (CutHeight, b0, CutId)) -> IO a1
    |
355 |     deleteRangeRocksDb (_getRocksDbCas cutHashesStore)
    |     ^^^^^^^^^^^^^^^^^^

src/Chainweb/CutDB.hs:359:9: error:
    Variable not in scope:
      compactRangeRocksDb
        :: RocksDbTable (CutHeight, BlockWeight, CutId) CutHashes
           -> (Maybe a2, Maybe (CutHeight, b1, CutId)) -> IO a3
    |
359 |         compactRangeRocksDb (_getRocksDbCas cutHashesStore)
    |         ^^^^^^^^^^^^^^^^^^^
```

Bisecting between 2.13 and 2.13.1 makes it look like #1342 introduced the error (I had to bisect skip #1385 because that had a *different* error, but I doubt that did it).

```
$ nix-info -m
 - system: `"x86_64-linux"`
 - host os: `Linux 5.10.40, NixOS, 21.05pre-git (Okapi)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.3.12`
```

Looks like `dep/chainweb-storage` needs to be updated. This patch fixes the above error:

```
diff --git a/dep/chainweb-storage/github.json b/dep/chainweb-storage/github.json
index 5b6bfe5a..c505cda9 100644
--- a/dep/chainweb-storage/github.json
+++ b/dep/chainweb-storage/github.json
@@ -2,6 +2,6 @@
   "owner": "kadena-io",
   "repo": "chainweb-storage",
   "private": false,
-  "rev": "1ee64539f9b6e27372caa14a1b4cbdf68736ed43",
-  "sha256": "1444cr558skrh98s3d3c3pvz3kd235n87wcibj45arbz7dji95vd"
+  "rev": "5bfe043ab11512b6eb4d625deaee0a79edc595df",
+  "sha256": "0sp4v1a80mlfzdpa7vi5wkpgwwqlnm9ss678vrkglpx6xh5xsq00"
 }
```

But then I get the error I had to skip from #1385:

```
[186 of 186] Compiling Chainweb.Chainweb ( src/Chainweb/Chainweb.hs, dist/build/Chainweb/Chainweb.o, dist/build/Chainweb/Chainweb.dyn_o )

src/Chainweb/Chainweb.hs:127:1: error:
    Could not find module ‘Network.Wai.Middleware.RequestSizeLimit’
    Perhaps you meant
      Network.Wai.Middleware.RequestLogger (from wai-extra-3.0.28)
      Network.Wai.Middleware.RequestLogger.JSON (from wai-extra-3.0.28)
    Use -v (or `:set -v` in ghci) to see a list of the files searched for.
    |
127 | import Network.Wai.Middleware.RequestSizeLimit
    | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Which appears to require a `wai` update, which probably requires a whole cascade of updates.

--- a/default.nix
+++ b/default.nix
@@ -149,7 +149,7 @@ pkgs.haskell.packages.${compiler}.developPackage {
 
       wai-extra = self.callHackageDirect {
         pkg = "wai-extra";
-        ver = "3.0.28";
+        ver = "3.1.2";
         sha256 = "1k470vbn2c852syj15m9xzfjnaraw6cyn35ajf2b67i01ghkshgw";
       } {};
 

--- a/default.nix
+++ b/default.nix
@@ -150,7 +150,7 @@ pkgs.haskell.packages.${compiler}.developPackage {
       wai-extra = self.callHackageDirect {
         pkg = "wai-extra";
         ver = "3.1.2";
-        sha256 = "1k470vbn2c852syj15m9xzfjnaraw6cyn35ajf2b67i01ghkshgw";
+        sha256 = "1cha6hvb071bknw25va07vg1sr5bg44q8fwz0nwa1886j9d4yrr7";
       } {};
 
       wai-app-static = doJailbreak (dontCheck (self.callHackageDirect {

--- a/dep/chainweb-storage/github.json
+++ b/dep/chainweb-storage/github.json
@@ -2,6 +2,6 @@
   "owner": "kadena-io",
   "repo": "chainweb-storage",
   "private": false,
-  "rev": "1ee64539f9b6e27372caa14a1b4cbdf68736ed43",
-  "sha256": "1444cr558skrh98s3d3c3pvz3kd235n87wcibj45arbz7dji95vd"
+  "rev": "5bfe043ab11512b6eb4d625deaee0a79edc595df",
+  "sha256": "0sp4v1a80mlfzdpa7vi5wkpgwwqlnm9ss678vrkglpx6xh5xsq00"
 }

--- a/src/Chainweb/Pact/Backend/Utils.hs
+++ b/src/Chainweb/Pact/Backend/Utils.hs
@@ -216,14 +216,14 @@ chainwebPragmas =
   [ "synchronous = NORMAL"
   , "journal_mode = WAL"
   , "locking_mode = NORMAL"
-  -- ^ changed from locking_mode = EXCLUSIVE to allow backups to run concurrently 
-  -- with Pact service operation. the effect of this change is twofold: 
-  --   - now file locks are grabbed at the beginning of each transaction; with 
-  --     EXCLUSIVE, file locks are never let go until the entire connection closes.
-  --     (see https://web.archive.org/web/20220222231602/https://sqlite.org/pragma.html#pragma_locking_mode)
-  --   - now we can query the database while another connection is open, 
-  --     taking full advantage of WAL mode.
-  --     (see https://web.archive.org/web/20220226212219/https://sqlite.org/wal.html#sometimes_queries_return_sqlite_busy_in_wal_mode)
+      -- changed from locking_mode = EXCLUSIVE to allow backups to run concurrently
+      -- with Pact service operation. the effect of this change is twofold:
+      --   - now file locks are grabbed at the beginning of each transaction; with
+      --     EXCLUSIVE, file locks are never let go until the entire connection closes.
+      --     (see https://web.archive.org/web/20220222231602/https://sqlite.org/pragma.html#pragma_locking_mode)
+      --   - now we can query the database while another connection is open,
+      --     taking full advantage of WAL mode.
+      --     (see https://web.archive.org/web/20220226212219/https://sqlite.org/wal.html#sometimes_queries_return_sqlite_busy_in_wal_mode)
   , "temp_store = MEMORY"
   , "auto_vacuum = NONE"
   , "page_size = 1024"

